OpenSim/Moco/MocoGoal/MocoContactImpulseTrackingGoal.h

```diff
@@ -19,18 +19,19 @@
  * -------------------------------------------------------------------------- */
 
 #include <OpenSim/Moco/osimMoco.h>
-#include "osimMocoImpulseTrackingGoalDLL.h"
+#include "osimMocoContactImpulseTrackingGoalDLL.h"
```

Comment:
This won't be necessary anymore because we are going to add this goal directly to the OpenSim API.